src/server/command_registry.cc

```diff
@@ -62,16 +63,65 @@ void CommandId::Invoke(CmdArgList args, ConnectionContext* cntx) const {
 
 CommandRegistry::CommandRegistry() {
   vector<string> rename_command = GetFlag(FLAGS_rename_command);
-
   for (string command_data : rename_command) {
     pair<string_view, string_view> kv = StrSplit(command_data, '=');
+
     auto [_, inserted] =
         cmd_rename_map_.emplace(AsciiStrToUpper(kv.first), AsciiStrToUpper(kv.second));
     if (!inserted) {
       LOG(ERROR) << "Invalid rename_command flag, trying to give 2 names to a command";
       exit(1);
     }
   }
+
+  config_registry.Register("rename_command", [this](const absl::CommandLineFlag& flag) {
+    auto res = flag.TryGet<vector<string>>();
+    if (!res)
+      return false;
+    return RenameCommands(*res);
+  });
+}
+
+bool CommandRegistry::RenameCommands(const std::vector<std::string>& rename_commands) {
+  auto apply_cb_on_all_commands = [](const std::vector<std::string>& commands,
+                                     std::function<bool(string_view, string_view)> cb) {
+    for (string command_data : commands) {
+      pair<string_view, string_view> kv = StrSplit(command_data, '=');
```

Comment:
Here and above, this will be unexpected behavior if the new value is expected to have `=` inside the name.
Maybe use `absl::MaxSplits('=', 1)` instead of just `'='`?